src/lib/dom.ts

```diff
@@ -23,9 +23,11 @@ interface MaybePolyfilledCe extends CustomElementRegistry {
 /**
  * True if the custom elements polyfill is in use.
  */
-export const isCEPolyfill = window.customElements != null &&
-    (window.customElements as MaybePolyfilledCe).polyfillWrapFlushCallback !==
-        undefined;
+export const isCEPolyfill = typeof window !== 'undefined' ?
```

Comment:
Since the false condition returns `false`, just use an and condition:

```suggestion
export const isCEPolyfill = typeof window !== 'undefined' &&
```

Reply:
lgtm